tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -469,25 +469,25 @@ procSuite "WakuNode - RLN relay":
           completionFut2.complete(true)
         if msg == wm3:
           completionFut3.complete(true)
-    
+
     # mount the relay handler for node2
     node2.subscribe((kind: PubsubSub, topic: DefaultPubsubTopic), some(relayHandler))
     await sleepAsync(2000.millis)
 
     await node1.publish(some(DefaultPubsubTopic), wm1)
-    await sleepAsync(10.seconds)
     await node1.publish(some(DefaultPubsubTopic), wm2)
-    await sleepAsync(10.seconds)
     await node1.publish(some(DefaultPubsubTopic), wm3)
-    
+
     let
       res1 = await completionFut1.withTimeout(10.seconds)
       res2 = await completionFut2.withTimeout(10.seconds)
       res3 = await completionFut3.withTimeout(10.seconds)
 
     check:
-      (res1 and res2 and res3) == true # all 3 are valid
-      node2.wakuRlnRelay.nullifierLog.len() == 1 # after clearing, only 1 is stored
+      res1 == true
+      res2 == false
```

Comment:
@rymnc Not sure why `rest2` was expected to complete. Since as stated above the second message "it will break the messaging rate limit" and that implies that the handler wont be triggered? So `res==false` would be expected isn't it?

Reply:
agree, oversight on my part